Archit/Source/Films/FilmsViewController.swift

```diff
@@ -77,13 +77,9 @@ class FilmsViewController: BaseViewController<FilmsController> {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(for: indexPath)
         let film = controller.films[indexPath.row]
-        cell.textLabel?.text = film.title
-        cell.detailTextLabel?.text = film.year
-        cell.imageView?.kf.setImage(with: film.poster, completionHandler: { (image, error, cache, url) in
-            if image != nil {
-                cell.layoutSubviews()
-            }
-        })
+        if let cell = cell as? FilmCell {
```

Comment:
Not necessary if you use [Reusable](https://github.com/AliSoftware/Reusable#3-dequeue-your-cells).